### DISCUSSION TITLE
Get Lazy Virtual Scroll to load more rows each time

### DISCUSSION THREAD

**Discussion by Amit-ga** [primefaces/community#1172] [PrimeReact] [answered]
Hello,
I have this primereact datatable with a lot of rows and columns, a very big table.
I enabled lazy virtual scroll, which has really improved the responsiveness of the table, and the filters and sorting are now working pretty fast. The only problem is that lazy scrolling only loads like 10 rows at a time, and waiting for the data to load in every scroll is exhausting, especially when there are more than 10,000 rows.
Is there any way to get it to load 100-150 rows each scroll?
Thanks!

**Comment by melloware** [comment, marked as answer]
Isn't it `itemSize` in the example: https://primereact.org/datatable/#lazy_virtualscroll

```js
virtualScrollerOptions={{ lazy: true, onLazyLoad: loadCarsLazy, itemSize: 46, delay: 200, showLoader: true, loading: lazyLoading, loadingTemplate }}
```
